src/site/_includes/js/eligibility.js

```diff
@@ -176,6 +176,20 @@ const cnst = {
       121650,
     ],
   },
+  hudvash: {
+    // https://siliconvalleyathome.org/resources/finding-affordable-housing/
+    // Last updated June 11, 2024
```

Comment:
I believe the HUD limits update every April 1, so you can say these numbers are "effective through 4/1/2025"